howto.md

```diff
@@ -197,6 +197,35 @@ It is possible to save both CPU time and memory by sharing the same resource acr
 
 The above sharing is safe because it is all happening in the same thread and subprocess (see below for details).
 
+By default, `smart_open` buffers the most recent part of a multipart upload in memory.
+The default part size is 50MB.
+If you're concerned about memory usage, then you have two options.
+The first option is to use smaller part sizes (e.g. 5MB, the lowest value permitted by AWS):
+
+```python
+import boto3
+from smart_open import open
+tp = {'min_part_size': 5 * 1024**2}
+with open('s3://bucket/key', 'w', transport_params=tp) as fout:
+    fout.write(lots_of_data)
+```
+
+This will split your upload into smaller parts.
+Be warned that AWS enforces a [limit](https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html) of a maximum of 10,000 parts per upload.
+
+The second option is to use a temporary file as a buffer instead.
+
+```python
+import boto3
+from smart_open import open
+with tempfile.NamedTemporaryFile() as tmp:
+    tp = {'writebuffer': tmp}
+    with open('s3://bucket/key', 'w', transport_params=tp) as fout:
+        fout.write(lots_of_data)
+```
+
+This option reduces memory usage at the expense of additional disk I/O (writing to a reading from a hard disk is slower).
```

Comment:
`This option reduces memory usage at the expense of additional disk I/O (writing to and reading from a hard disk is slower).`

Reply:
I'll fix this on develop.

Reply:
Thanks. Super minor, sorry I noticed it only now.